buildSrc/src/main/kotlin/dependencies/Dependencies.kt

```diff
@@ -4,36 +4,33 @@
 
 @file:Suppress("PackageDirectoryMismatch", "SpellCheckingInspection")
 
-@Suppress("unused")
-object Versions {
-    const val kotlin = "1.3.50" // Don't forget to update in buildSrc/build.gradle.kts too!
-}
-
 /**
  * Nested objects have a corresponding property to allow usage from groovy based gradle scripts.
  */
 @Suppress("unused")
 object Libs {
-    const val junit = "junit:junit:4.12"
+    const val kotlinVersion = Versions.org_jetbrains_kotlin
+    
+    const val junit = "junit:junit:" + Versions.junit
     const val roboElectric = "org.robolectric:robolectric:4.2"
-    const val timber = "com.jakewharton.timber:timber:4.7.1"
-    const val stetho = "com.facebook.stetho:stetho:1.5.0"
+    const val timber = "com.jakewharton.timber:timber:" + Versions.timber
+    const val stetho = "com.facebook.stetho:stetho:" + Versions.stetho
 
     val kotlin = Kotlin
     val kotlinX = KotlinX
     val androidX = AndroidX
     val google = Google
 
     object Kotlin {
-        const val stdlibJdk7 = "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${Versions.kotlin}"
-        const val testJunit = "org.jetbrains.kotlin:kotlin-test-junit:${Versions.kotlin}"
+        const val stdlibJdk7 = "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+        const val testJunit = "org.jetbrains.kotlin:kotlin-test-junit:$kotlinVersion"
     }
 
     object KotlinX {
         val coroutines = Coroutines
 
         object Coroutines {
-            private const val version = "1.3.0"
+            private const val version = Versions.org_jetbrains_kotlinx
```

Comment:
This will clash with other kotlinx libraries with different versions like serialization, immutable.collections, html, file-io… Can buildSrcVersions still share the version, using the artifact name prefix, or could it be a new feature?